src/librustc_yk_sections/emit_tir.rs

```diff
@@ -10,14 +9,17 @@
 //! This module converts MIR into Yorick TIR (Tracing IR).
 //! Note that we preserve the MIR block structure when lowering to TIR.
 //!
+//! Note also that TIR is currently untyped.
```

Comment:
I don't think this is worth stating: TIR is many things (and isn't many other things), and that'll be in flux for a while. This particular property seems to me only one of many.

Reply:
OK. TBH I don't think we will get far without types anyway.